src/accounts/context/userAccount.tsx

```diff
@@ -70,6 +77,24 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
     }
   }
 
+  const handleCancelAccount = useCallback(async () => {
+    try {
+      const resp = await postCancel({})
+
+      if (resp.status !== 204) {
+        throw new Error(resp.data.message)
+      }
+
+      if (!isFlagEnabled('trackCancellations')) {
+        history.push(`/logout`)
+      }
```

Comment:
why are users getting logged out only if we're tracking cancellations?

Reply:
if i remember correctly, if the trackCancellations is turned on, we are logging out using a different mechanism.